h5io/_h5io.py

```diff
@@ -326,6 +333,7 @@ def _triage_write(
             where + ".csr_matrix_indptr",
             cleanup_data=cleanup_data,
             slash=slash,
+            use_state=use_state,
```

Comment:
Is there a case where the `indptr` are required to be an object which needs `__getstate__()` to be stored in the HDF5 file?